static/app/views/onboarding/targetedOnboarding/components/welcomeBackground.tsx

```diff
@@ -0,0 +1,320 @@
+import * as React from 'react';
+import {keyframes} from '@emotion/react';
+import styled from '@emotion/styled';
+import {motion} from 'framer-motion';
+import random from 'lodash/random';
+
+import testableTransition from 'sentry/utils/testableTransition';
+
+// Using framer motion for this loop for the nice physics
```

Comment:
Do we need this comment really?

Reply:
Well this is copy pasted from your version https://github.com/getsentry/sentry/blob/ba15b6285fd8d190e249f3acc3fe42927d622f21/static/app/views/onboarding/components/welcomeBackground.tsx#L9

Reply:
hahaha it was me :) ok nvm